statefun-flink/statefun-flink-io-bundle/src/main/java/org/apache/flink/statefun/flink/io/kafka/binders/ingress/v1/RoutableKafkaIngressDeserializer.java

```diff
@@ -45,7 +45,7 @@ public RoutableKafkaIngressDeserializer(Map<String, RoutingConfig> routingConfig
   public Message deserialize(ConsumerRecord<byte[], byte[]> input) {
     final String topic = input.topic();
     final byte[] payload = input.value();
-    final byte[] key = requireNonNullKey(input.key());
```

Comment:
hi, what do you think about adding the ability to skip the record with null key and make this functionality configurable like:
```
kind: io.statefun.kafka.v1/ingress
spec:
  id: localhost:9092
  consumerGroupId: test-group-dev
  startupPosition:
    type: earliest
  properties:
    - isolation.level: read_committed
    - security.protocol: SSL
    - ssl.truststore.location: /tmp/flink/jks/truststore.jks
    - ssl.truststore.password: changeit
    - ssl.keystore.location: /tmp/flink/jks/kafka_keystore.jks
    - ssl.keystore.password: test
    - ssl.key.password: test
  topics:
    - topic: test-dev
      skipNullKeyRecords: true
      valueType: ua.Test
      targets:
        - ua.Test
```

Reply:
That might work. The `NoOpDeserliazer` just returns `null`, which could be done here as well. https://github.com/apache/flink-statefun/blob/accd75ea0109845c4b4c0ddd74021147af1439d4/statefun-kafka-io/src/test/java/org/apache/flink/statefun/sdk/kafka/KafkaIngressBuilderTest.java#L166